packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -514,10 +514,8 @@ class HotRunner extends ResidentRunner {
     String reason,
     bool benchmarkMode = false,
   }) async {
-    if (!_isPaused()) {
```

Comment:
I could not find a good justification for keeping the pause check

Reply:
https://github.com/flutter/flutter/pull/9733 fixed https://github.com/flutter/flutter-intellij/issues/969, make sure removing this check doesn't regress that.

Reply:
Sorry, should have been more clear here:

We have a hot reload test in integration.shard that performs a hot restart while paused and that still passes with this removed. I've also verified locally that this listViews RPC returns successfully while paused:

```
An Observatory debugger and profiler on Pixel 4 is available at: http://127.0.0.1:50743/Nb1i5jI30FQ=/


Performing hot reload...
The application is paused due in the debugger; interface might not update.
Reloaded 0 of 660 libraries in 107ms.

Performing hot reload...
The application is paused due in the debugger; interface might not update.
Reloaded 0 of 660 libraries in 100ms.
```

Reply:
This was most likely a bug in the implementation of listVews. I can do some more research to try and figure out when it was fixed, but for now we should have good coverage